indexing-service/src/test/java/io/druid/indexing/common/task/AppenderatorDriverRealtimeIndexTaskTest.java

```diff
@@ -491,13 +504,19 @@ public void testMaxRowsPerSegment() throws Exception
     awaitHandoffs();
 
     for (DataSegment publishedSegment : publishedSegments) {
-      Optional<Map.Entry<SegmentDescriptor, Pair<Executor, Runnable>>> optional = handOffCallbacks.entrySet().stream()
-                                                                                                  .filter(e -> e.getKey().equals(new SegmentDescriptor(
-                                                                                                      publishedSegment.getInterval(),
-                                                                                                      publishedSegment.getVersion(),
-                                                                                                      publishedSegment.getShardSpec().getPartitionNum()
-                                                                                                  )))
-                                                                                                  .findFirst();
+      Optional<Map.Entry<SegmentDescriptor, Pair<Executor, Runnable>>> optional = handOffCallbacks
```

Comment:
Same

Reply:
I'm not going to change logic in this PR